cmd/init.go

```diff
@@ -122,41 +134,122 @@ func executeInit(devPath string, overwrite bool, language string, workDir string
 		}
 	}
 
-	analytics.TrackInit(true)
+	analytics.TrackInit(true, language)
 	return nil
 }
 
-func askForImage(language, defaultImage string) string {
-	var image string
-	fmt.Printf("Recommended image for development with %s: %s\n", language, log.BlueString(defaultImage))
-	fmt.Printf("Which docker image do you want to use for your development container? [%s]: ", defaultImage)
-	_, err := fmt.Scanln(&image)
-	fmt.Println()
+func getDeployment(namespace string) (*appsv1.Deployment, string, error) {
```

Comment:
can split this in two functions? one that gets the deployment / container, and another one that asks the user for info?

Reply:
Also, you need to stop the spinner before showing text or the terminal can start behaving strangely.